core/workflow-java/src/main/java/org/apache/syncope/core/workflow/java/AbstractUserWorkflowAdapter.java

```diff
@@ -78,6 +78,10 @@ public String getPrefix() {
         return doUpdate(userDAO.authFind(userPatch.getKey()), userPatch);
     }
 
+    @Override
+    public abstract WorkflowResult<String> doRequestCertify(final User user);
```

Comment:
What's the purpose of this abstract method? It is declared in the interface and implemented in the concrete subclasses, it could be safely removed.